pyrebase/pyrebase.py

```diff
@@ -553,9 +555,8 @@ def start(self):
     def start_stream(self):
         self.sse = ClosableSSEClient(self.url, session=self.make_session(), build_headers=self.build_headers)
         for msg in self.sse:
-            msg_data = json.loads(msg.data)
-            # don't return initial data
-            if msg_data:
+            if msg:
+                msg_data = json.loads(msg.data)
```

Comment:
You will need to check whether msg_data is something as well, as "msg.data" can be "null", which is a message, but evaluates to python "None", which will crash on line 560 when you try to assign to it.
Here's what we've done: https://github.com/comoyo/Pyrebase/commit/81ed523407ab0f3ceb7f6168bcc1af23ca36938e.